app/views/questionnaire.py

```diff
@@ -260,7 +312,7 @@ def _is_skipping_to_the_end(routing_path, block, current_location):
         block['type'] in END_BLOCKS
 
 
-def submit_answers(routing_path, eq_id, form_type, collection_id):
+def submit_answers(routing_path, eq_id, form_type, collection_id, block_id):
```

Comment:
Going to refactor the way we define if we show the submitted responses or not. This will come out as part of that. In future we'll define it within an object in the schema so it won't be dependent on having a summary page.